test/test_pipeline/test_metrics.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+import torch
+
+from autoPyTorch.pipeline.components.training.metrics.base_metric import autoPyTorchMetric
+from autoPyTorch.pipeline.components.training.metrics.utils import get_metrics
+from autoPyTorch.pipeline.components.training.metrics.utils import get_supported_metrics
+
+
+class MetricsTest(unittest.TestCase):
```

Comment:
When you add an attribute to tell if a loss is better the higher it gets, or the lower it gets, we should add a test for this.